0_RootFS/Rootfs/bundled/conf/profile.d/bb_utils.sh

```diff
@@ -62,15 +62,40 @@ qfind() {
     find "$@" 2>/dev/null
 }
 
-# Function to install license files
-install_license () {
-    if [ $# -eq 0 ]; then
-        echo "Usage: install_license license_file1.txt [license_file2.md, license_file3.rtf, ...]" >&2
-        exit 1
+# Function to automatically install license files at the end of the build
+auto_install_license () {
+    if [[ -z "$(qfind "${prefix}/share/licenses/${SRC_NAME}" -mindepth 1)" ]]; then
```

Comment:
I would rather just `-d ${prefix}/share/licenses/${SRC_NAME}`

Reply:
The idea of the test was to check if there are actually files inside the directory, not just if the dir is there.  Do you think that just checking the dir is sufficient?

Reply:
I think the auto-install should fail out very aggressively; if someone tries to install a specific license and it doesn't work for some reason, the very fact that they tried (and created an empty directory) should be enough to scare off the auto-install routine, IMO.  We should only auto-install if absolutely no action to install a license has been taken.